samples/constitutions/default/actions.js

```diff
@@ -914,6 +914,19 @@ const actions = new Map([
       }
     ),
   ],
+  [
+    "add_executor_node_code",
+    new Action(
+      function (args) {
+        checkType(args.executor_code_id, "string", "executor_code_id");
+      },
+      function (args) {
+        const codeId = ccf.strToBuf(args.executor_code_id);
+        const ALLOWED = ccf.jsonCompatibleToBuf("AllowedToJoin");
```

Comment:
`AllowedToJoin` makes sense but it is the same enum used for nodes to join the CCF service. For clarify, I think we should create a new `CodeStatus` enum and use `AllowedToExecute` or something similar to avoid ambiguity.

Reply:
Done

Reply:
I reverted this change for a couple of reasons a) This means the API schema will change and we don't really want this in 3.x and b) Even though the enum is same, since it sits in a different table it's distinguishable

Reply:
I think we can go one step further, and usually a totally custom enum/string for this value (since, as you say, it exists in a separate table). So creating an `ExecutorCodeStatus` enum, with a single value `AllowedToExecute`, only visible to this application, and using that in place of `ccf::CodeStatus`. That means if `CodeStatus` ever gets new values (eg - `AllowedToJoinAsLearner` or something), we don't ever accidentally consider that a valid status for an Executor.

Reply:
Sounds reasonable, done.